charts/icm-as/tests/newrelic_test.yaml

```diff
@@ -30,10 +30,45 @@ tests:
     set:
       newrelic.enabled: true
       newrelic.license_key: "secret-license-test"
-    template: config-newrelic-yaml-cm.yaml
+    template: templates/config-newrelic-yaml-cm.yaml
     asserts:
       - isKind:
           of: ConfigMap
       - matchRegex:
           path: data["newrelic.yml"]
           pattern: .*(secret-license-test).*
+
+  - it: newrelic app_name from operationalContext (default values)
+    release:
+      name: icm-as
+    chart:
+      version: 0.8.15
+    values:
+      - ../values.yaml
+    set:
+      newrelic.enabled: true
+    template: templates/config-newrelic-yaml-cm.yaml
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: data["newrelic.yml"]
+          pattern: ".*(app_name : \"n_a-prd-icm-standalone\").*"
```

Comment:
please declare an environment name to show that "prd" is the environment name not the type.